overlay/overlay.nix

```diff
@@ -104,6 +110,16 @@ in
       ];
     });
 
+    modemmanager = super.modemmanager.overrideDerivation (super: {
```

Comment:
This requires modemmanager 1.18+, which is not in master yet. NixOS/nixpkgs#141684